### ISSUE TITLE
Does not work out of the box if pdo_sqlite is not installed

### ISSUE THREAD

**Issue by yahuarkuntur** [symfony/symfony-demo#29]
I just downloaded the demo app (git clone) then installed the dependencies with composer and the project is installed with no warnings or errors.

When trying to load the fixtures, the doctrine command fails, since I dont have installed the SQLite driver in my dev box.

I wonder if this "dependency" can be set directly in composer or update the installation docs?

Best regards,

**Comment by javiereguiluz** [comment]
@yahuarkuntur thanks for reporting this issue. I find it strange to disable SQLite extension in a development machine, but we should take care of that to avoid issues for some users.

According to Composer documentation, we can state the required PHP extensions as follows: 

```json
"require": {
    "php"        : ">=5.3.3",
    "ext-sqlite" : "*",
}
```

Is this correct? Is this enough? Should we also add the `pdo` extension dependency?

**Comment by yahuarkuntur** [comment]
@javiereguiluz I am not sure if `ext-sqlite` or `ext-pdo-sqlite` is better. But both seem to warn me when issuing a composer update...

**Comment by stof** [comment]
Given we use pdo_sqlite, ``ext-pdo-sqlite`` is better. You can technically enable them separately (even though apt packages are bundling them together).

Adding the pdo dependency would be a good idea too indeed.

@javiereguiluz given that people will download the demo with the installer, we should also add a check in the symfony requirements if we can. Relying on the composer failure will not help if they don't run composer.

**Comment by javiereguiluz** [comment]
Fixed in #32
